tools/config_updater.py

```diff
@@ -696,6 +740,91 @@ def ManageBinaries(config=None, token=None):
   print "Please restart the service for it to take effect.\n\n"
 
 
+def SuggestedConfiguration():
```

Comment:
Yeah, but all of this is pre-production deployment config and can wait until after you've got it up and running and tried things out. The install process is long enough and full of tons of prompts. I think we should beg off anything that isn't essential to getting a working install until later. These are the sorts of config changes you should make once you have a config under version control and are working on getting it into production shape.